src/java.base/share/classes/sun/net/www/protocol/file/FileURLConnection.java

```diff
@@ -40,6 +34,11 @@
 import java.util.*;
 import java.text.SimpleDateFormat;
 
+/**
+ * Open an file input stream given a URL.
```

Comment:
```suggestion
 * Open a file input stream given a URL.
```

Reply:
OK, I'll accept this as a minor typo mixup, that does not in itself need a CSR. But generally, it is not a goal to fix issues in the content of doc comments.